workshop.wwwapi/Data/DatabaseContext.cs

```diff
@@ -10,24 +9,39 @@ public class DatabaseContext : DbContext
         private string _connectionString;
         public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options)
         {
-            var configuration = new ConfigurationBuilder().AddJsonFile("appsettings.json").Build();
+            var configuration = new ConfigurationBuilder().AddJsonFile("appsettings.Development.json").Build();
             _connectionString = configuration.GetValue<string>("ConnectionStrings:DefaultConnectionString")!;
             this.Database.EnsureCreated();
         }
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             //TODO: Appointment Key etc.. Add Here
-            
+            modelBuilder.Entity<Appointment>().HasKey(e => new { e.PatientId, e.DoctorId });
```

Comment:
probably want to include the datetime as part of the PK so that you can book same person on same date with same doctor

Reply:
but don't change this, this is also ok